packages/commonwealth/client/scripts/views/pages/user_dashboard/TrendingCommunitiesPreview/TrendingCommunitiesPreview.tsx

```diff
@@ -10,6 +10,12 @@ export const TrendingCommunitiesPreview = () => {
 
   const sortedCommunities = app.config.chains
     .getAll()
+    .filter((community) => {
+      const name = community.name.toLowerCase();
+      return (
+        !['"', '>', '<'].includes(name[0]) && !['"', '>', '<'].includes(name[1])
+      );
+    })
```

Comment:
This is awesome but we're only doing this on the frontend. I assume we're doing this on the backend as well?

Reply:
If we are then just resolve.

Reply:
@burtonator Yes, this was simply for the frontend to not show any of those communities within the Trending Communities column. I'll make a new ticket to fix this on the backend. Thanks for the suggestion.